src/decisionengine/framework/taskmanager/ProcessingState.py

```diff
@@ -54,7 +54,8 @@ def __init__(self, state=State.BOOT):
         self._cv = multiprocessing.Condition()
         self._lock = multiprocessing.RLock()
         self._state = multiprocessing.Value('i', allowed_state.value)
-        self.logger = logging.getLogger()
+        self.logger = structlog.getLogger()
```

Comment:
Which logger is this one getting? The one from the channel? Or could be the decision_engine one?

Reply:
This is getting the decisionengine logger. I have put the name in to make it explicit.